roles/operator_bundle_scorecard_tests/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Run operator scorecard test"
+  block:
+  - name: "Set facts for the Operator groups"
+    set_fact:
+      OO_BUNDLE: "{{ OO_BUNDLE }}"
+
+  # incase of failure the following task fails
+  # we will be ignoring errors to display the results
+  - name: "Generate namespace name if not provided"
+    shell: "{{ lookup('template', 'optera') }}"
```

Comment:
I'd advise to use the `file` type for lookup and moving the `optional-operators-cvp-common-scorecard-commands.sh` script from  `templates` folder to `files`, since the `template` type could cause issues if there are any Ansible-readable strings in the script. 

Also I'm guessing that `optera` is a typo and should be the script's filename.